cinempc/DroneAndCameraState drone_state
cinempc/TargetState target_state
bool found

